src/pages/Dashboard/Actions/AddDoThis.vue

```diff
@@ -0,0 +1,676 @@
+          </v-btn>
+        </template>
+
+        <v-card width="30vW">
+          <v-card-text
+            ><div class="mb-2"
+              >The default message varies depending on what type of action you
+              attach the config to.
+            </div>
+            <div>
+              For a state change event the message would be:
+              <span class="font-weight-light"
+                >"Run `{flow_run_name}` of flow `{flow_name}` entered state
+                `{state}` with " "message `{state_message}`. See {flow_run_link}
+                for more details."</span
+              ></div
+            >
+            <div
+              >For a flow SLA event the default message would be:
+              <span class="font-weight-light"
+                >"Run `{flow_run_name}` (`{flow_run_id}`) of flow `{flow_name}`
+                failed `{kind}` " "SLA (`{flow_sla_config_id}`) after
+                {duration_seconds} seconds. " "See {flow_run_link} for more
+                details."</span
+              ></div
+            >
+            <div
+              >For an agent SLA event, the default message would be:
+              <span class="font-weight-light"
+                >"Agents sharing the config {agent_config_id} have failed the
+                minimum healthy " "count of {sla_min_healthy}. The following
+                agents are unhealthy: {agent_ids}"</span
+              ></div
+            ></v-card-text
+          >
+        </v-card>
+      </v-menu>
+      <!-- <v-checkbox v-model="bothMessages" class="mt-0 no-top" dense>
+        <template #label
+          ><span class="body-2"
+            >Add this message <span class="font-weight-bold"> and</span> the
+            default message.</span
+          ></template
+        >
+      </v-checkbox> -->
+      <v-text-field
+        v-model="messageText"
+        class="pt-0"
+        @keydown.enter="saveMessage"
+      />
+    </v-card-text>
+    <v-card-text v-else-if="step === 'openToConfig'">
+      <v-row v-if="messageType.type === 'SLACK_WEBHOOK'" class="py-3">
+        <v-col
+          v-for="name in secretNames"
+          :key="name"
+          cols="6"
+          sm="3"
+          lg="2"
+          class="pa-1"
+        >
+          <div
+            v-ripple
+            class="chip-small d-flex align-center justify-start px-2 cursor-pointer user-select-none"
+            :class="{ active: secretName === name }"
+            @click="selectSecret(name)"
+            ><div class="text-truncate">
+              {{ name }}
+            </div></div
+          >
+        </v-col>
+      </v-row>
+      <!-- <v-text-field
+        v-if="messageType.type === 'SLACK_WEBHOOK'"
+        v-model="messageConfigTo"
+        :label="messageConfigLabel"
+        :rules="[rules[messageType.type]]"
+        validate-on-blur
+        :error-messages="errorMessage"
+        @keyup.enter="saveConfig"
+      ></v-text-field> -->
+      <div v-else-if="isPagerDuty">
```

Comment:
![Screen Shot 2021-03-22 at 5 35 52 PM](https://user-images.githubusercontent.com/3011231/112061597-17937780-8b35-11eb-8e9c-61cc7025be11.png)
I think pagerduty's missing a step? (to whom?)